src/server/lib/renderer.tsx

```diff
@@ -140,6 +142,7 @@ export const renderer: <P extends RoutePaths>(
 
 	return `
     <!DOCTYPE html>
+    <!--Request ID: ${clientState.shortRequestId}-->
```

Comment:
I'd leave a space on either side, so it can't easily be mistake for another tag.
```suggestion
    <!-- Request ID: ${clientState.shortRequestId} -->
```